docker/Dockerfile

```diff
@@ -46,8 +46,10 @@ RUN echo "/dmod/shared_libs/" >> /etc/ld.so.conf.d/ngen.conf && ldconfig -v
 COPY   HelloNGEN.sh /ngen/HelloNGEN.sh
 RUN chown -R root /dmod /ngen /root
 RUN chmod a+x /dmod/bin/* /ngen/HelloNGEN.sh /root
-RUN rm -rf tmp var opt 
```

Comment:
Any reason to keep these directories in? initially we got rid of them to keep the image small.

Reply:
I need to double check, but I think because they're included in the previous layer (rocky9.1) removing them doesn't actually reduce the size of the final output image. It's handy to have them so dnf works to install debug packages.

Reply:
Just tested it and the image is 2.24GB with and without those directories, we could move to rocky minimal to make it smaller, but it's only a 15mb space save so probably not worth the time it would take to migrate.